.babelrc

```diff
@@ -1,4 +1,4 @@
 {
-  "presets": ["env"],
-  "plugins": ["transform-object-rest-spread"]
+  "presets": ["@babel/preset-env"],
+  "plugins": ["@babel/plugin-proposal-object-rest-spread"]
```

Comment:
I think rest-spread plugin is already included in preset-env as it's in ES2018.

https://babeljs.io/docs/en/plugins#es2018

Reply:
Thanks for pointing out. Yap, I've tested and indeed seems like we don't need it. Removed.